src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -764,6 +765,22 @@ else if (hand == EquipmentSlot.OFF_HAND)
 			EventValues.registerEventValue(WorldBorderCenterChangeEvent.class, Location.class, WorldBorderCenterChangeEvent::getNewCenter);
 			EventValues.registerEventValue(WorldBorderCenterChangeEvent.class, Location.class, WorldBorderCenterChangeEvent::getOldCenter, EventValues.TIME_PAST);
 		}
+
+		EventValues.registerEventValue(VillagerCareerChangeEvent.class, VillagerCareerChangeEvent.ChangeReason.class, VillagerCareerChangeEvent::getReason);
```

Comment:
do you think it's safe to add a PAST event value that uses Villager#getProfession?